slick-testkit/src/main/scala/com/typesafe/slick/testkit/tests/OldPlainSQLTest.scala

```diff
@@ -110,15 +110,55 @@ class OldPlainSQLTest extends TestkitTest[JdbcTestDB] {
     println("User for ID 2: "+res)
     assertEquals(User(2,"guest"), res)
 
-    val s1 = sql"select id from USERS where name = ${"szeiger"}".as[Int]
-    val s2 = sql"select id from USERS where name = '#${"guest"}'".as[Int]
-    assertEquals("select id from USERS where name = ?", s1.getStatement)
+    val s1 = sql"select id from USERS where name = '${"szeiger"}'".as[Int]
+    val s2 = sql"select id from USERS where name = '${"guest"}'".as[Int]
+    assertEquals("select id from USERS where name = 'szeiger'", s1.getStatement)
```

Comment:
Why does the interpolation behave differently? We shouldn't change anything here. In particular, the default without any qualifier before the `$` should use bind variables and safeguard against SQL injection.